app/models/repository.rb

```diff
@@ -169,7 +201,7 @@ def self.create_or_update!(repo)
     end
 
     # Finally remove the tags that are left and return the repo.
-    repository.tags.where(name: to_be_deleted_tags).find_each(&:delete_and_update!)
+    repository.tags.where(name: to_be_deleted_tags).find_each { |t| t.delete_and_update!(portus) }
```

Comment:
This will mark the deletion event as to be generated by the `portus` user. However into your screenshot the event is assigned to the right user.

How is that happening?